src/components/richLink.tsx

```diff
@@ -46,40 +46,52 @@ const richLinkStyles = (format: Format): SerializedStyles => {
 	const backgroundColor =
 		format.design === Design.Comment ? neutral[86] : neutral[97];
 
-	const { kicker: newsKicker } = getThemeStyles(Pillar.News);
-	const { kicker: opinionKicker } = getThemeStyles(Pillar.Opinion);
-	const { kicker: sportKicker } = getThemeStyles(Pillar.Sport);
-	const { kicker: cultureKicker } = getThemeStyles(Pillar.Culture);
-	const { kicker: lifestyleKicker } = getThemeStyles(Pillar.Lifestyle);
+	const { kicker: newsKicker, inverted: newsInverted } = getThemeStyles(
+		Pillar.News,
+	);
+	const { kicker: opinionKicker, inverted: opinionInverted } = getThemeStyles(
+		Pillar.Opinion,
+	);
+	const { kicker: sportKicker, inverted: sportInverted } = getThemeStyles(
+		Pillar.Sport,
+	);
+	const { kicker: cultureKicker, inverted: cultureInverted } = getThemeStyles(
+		Pillar.Culture,
+	);
+	const {
+		kicker: lifestyleKicker,
+		inverted: lifestyleInverted,
+	} = getThemeStyles(Pillar.Lifestyle);
 
 	return css`
 		background: ${backgroundColor};
-		padding: ${remSpace[3]};
+		padding: ${remSpace[3]} ${remSpace[3]} ${remSpace[2]};
 		border-top: solid 1px ${neutral[60]};
 		transition: all 0.2s ease;
 
 		&.js-news {
-			${richLinkPillarStyles(newsKicker)}
+			${richLinkPillarStyles(newsKicker, newsInverted)}
 		}
 
 		&.js-opinion {
-			${richLinkPillarStyles(opinionKicker)}
+			${richLinkPillarStyles(opinionKicker, opinionInverted)}
 		}
 
 		&.js-sport {
-			${richLinkPillarStyles(sportKicker)}
+			${richLinkPillarStyles(sportKicker, sportInverted)}
 		}
 
-		&.js-culture {
-			${richLinkPillarStyles(cultureKicker)}
+		&.js-culture,
+		&.js-arts {
```

Comment:
Hold on, how does this class get applied to the element? There is no "arts", it's the old name for "culture".

Reply:
![Screenshot 2021-06-24 at 17 09 58](https://user-images.githubusercontent.com/12860328/123296964-03ce2780-d50f-11eb-977d-58e817bf0a05.png)

Very good question ... I was puzzled by this too hence my commit message. I believe the class is added via this https://github.com/guardian/apps-rendering/blob/79841c991c6089a781ffcf970476f6abae17a182/src/client/article.ts#L456 
Which makes me think we're getting the data wrong from MAPI/CAPI?

Reply:
If that's just proxying the `pillar` from CAPI then it will give you arts. It's still called that in the CAPI response for legacy reasons. We'll want to map it across to `js-culture` if it discovers `arts` (could do that here or at the MAPI level).

Reply:
I'd vote to do that in MAPI, because if we did in here then we have a situation where MAPI is serving legacy stuff, and code here that is trying to fix that. If we fix it in MAPI, we have a more "sound" content flow.